vispy/gloo/preprocessor.py

```diff
@@ -45,7 +45,7 @@ def replace(match):
                 logger.critical('"%s" not found' % filename)
                 raise RuntimeError("File not found", filename)
             text = '\n// --- start of "%s" ---\n' % filename
-            text += remove_comments(open(path).read())
+            text += open(path).read()
```

Comment:
Yeah I guess they're not leaked, you just get annoying warnings in some circumstances. I've gotten them on Linux before, and in the past I've had problems with Travis logs being full of them on some repos. You can leave it if you want and we can change it later if need be. Personally I prefer the explicitness of `with` in any case.